### ISSUE TITLE
Phase I deploy

### ISSUE THREAD

**Issue by ddohler** [azavea/echo-locator#361]
Once the Phase 1 Milestone is complete, we should do a deploy.

* As part of this we should update the domain to be a TBD subdomain of echosearch.org (if possible, see next item).
* If possible, also add a CNAME from echolocator.org to the new subdomain. If that's not possible, then DO NOT change the domain name because it will break the "Login" button on https://echosearch.org.

**Comment by ddohler** [comment]
Here's my deployment checklist:

- [x] Backed up existing production site by running `AWS_PROFILE=echo-locator aws s3 sync s3://echo-locator-production-site-us-east-1/ .`
- [x] Double-checked contents of config file: `AWS_PROFILE=echo-locator aws s3 sync s3://echo-locator-production-config-us-east-1 .` (and then inspect `taui/env.yml`)
- [x] Merge release branch to master
- [x] Images get synced separately, no need to update them now.
- [x] Merge master -> develop

**Comment by ddohler** [comment]
The new code is up and available on production! Closing this and opening a new issue to deal with the DNS since that info wasn't available.

![Screen Shot 2022-05-04 at 10.08.14.png](https://images.zenhubusercontent.com/5ca60a56f624253ca76d1781/fc1a7db3-05e8-4b0f-9693-e9ac4d484910)
